Resources/Prototypes/Entities/Mobs/NPCs/funnyayadoll.yml

```diff
@@ -0,0 +1,89 @@
+- type: entity
+  name: poltergeist
+  id: Mobayadoll
```

Comment:
I understand the source material it's inspired by, but a more descriptive entity ID could be helpful, such as indicating it's a poltergeist doll.